browser/search_engines/tor_window_search_engine_provider_service.cc

```diff
@@ -15,8 +16,8 @@
 TorWindowSearchEngineProviderService::
 TorWindowSearchEngineProviderService(Profile* otr_profile)
     : SearchEngineProviderService(otr_profile) {
-  DCHECK(otr_profile->IsTorProfile());
-  DCHECK(IsGuestProfile(otr_profile));
+  DCHECK(tor::IsTorProfile(otr_profile));
+  DCHECK(brave::IsOTRGuestProfile(otr_profile));
```

Comment:
this check is not needed and won't be relevant when tor profiles are not based on guest profiles